src/main_loop.rs

```diff
@@ -119,13 +123,12 @@ impl MainLoop {
             );
 
             let mixer = (self.audio_setup.mixer)();
-            let audio_filter = mixer.get_audio_filter();
             let backend = self.audio_setup.backend;
             let audio_device = self.audio_setup.audio_device.clone();
             let (player, mut event_channel) = Player::new(
                 self.player_config.clone(),
                 session.clone(),
-                audio_filter,
+                Box::new(NoOpVolume),
```

Comment:
I think, this needs to be 
```suggestion
                mixer.get_soft_volume(),
```
for `SoftVol` to work. (Otherwise, the `Player` has no idea, what the multiplication factor for the audio samples.)